org.jacoco.core.test/src/org/jacoco/core/internal/flow/LabelInfoTest.java

```diff
@@ -90,6 +91,42 @@ public void testMultiTarget3() {
 	}
 
 	@Test
+	public void testMethodInvocationLine() {
+		LabelInfo.setMethodInvocationLine(label);
+		assertTrue(LabelInfo.isMethodInvocationLine(label));
+	}
+
+	@Test
+	public void testNeedsProbe() {
+		testNeedsProbe(false, false, false, false);
+		testNeedsProbe(true, false, false, false);
+		testNeedsProbe(false, true, false, false);
+		testNeedsProbe(true, true, false, false);
+		testNeedsProbe(false, false, true, false);
+		testNeedsProbe(true, false, true, true);
+		testNeedsProbe(false, true, true, true);
+		testNeedsProbe(true, true, true, true);
```

Comment:
Any other proposal? Alternatively I can create scenario tests for this (as already used in this test class).